config/core/test-configs-riscv.yaml

```diff
@@ -0,0 +1,225 @@
+default_filters:
+
+  test_plans:
+
+    - blocklist: &kselftest_defconfig_filter
+        defconfig: ['kselftest']
+
+
+test_plans:
+
+  kselftest-all_riscv: &kselftest_riscv
+    base_name: kselftest
+    rootfs: debian_sid-kselftest_nfs
+    pattern: 'kselftest/{category}-{method}-{protocol}-{rootfs}-kselftest-template.jinja2'
+    filters:
+      - passlist: {defconfig: ['kselftest']}
+
+  kselftest-all_riscv_qemu: &kselftest_riscv_qemu
+    rootfs: debian_sid-kselftest_ramdisk
+    pattern: 'kselftest/{category}-{method}-{rootfs}-kselftest-template.jinja2'
+    filters:
+      - passlist: {defconfig: ['kselftest']}
+      - passlist: &qemu_labs_filter
+          lab:
+            - 'lab-baylibre'
+            - 'lab-broonie'
+            - 'lab-collabora'
+            - 'lab-collabora-staging'
```

Comment:
Yes this one is needed as KernelCI QEMU jobs are required to run in Docker which involves some support in the labs and not all labs have it in place.  The list is a bit old actually so maybe more labs are now able to run Docker QEMU jobs.